jupyterhub/apihandlers/users.py

```diff
@@ -193,6 +197,52 @@ def delete(self, name):
         status = 202 if user.stop_pending else 204
         self.set_status(status)
 
+
+class UserCreateMultiServerAPIHandler(APIHandler):
+    """Create multi single-user server
+    
+    This handler should be used when c.JupyterHub.allow_multiple_servers = True
+    """
+    @gen.coroutine
+    @admin_or_self
+    def post(self, name):
+        user = self.find_user(name)
```

Comment:
@barrachri it was not possible before, but it is now due to admin services. This can add

```python
if user is None:
    raise HTTPError(404, "No such user %r" % name)
```